<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectWindow</class>
 <widget class="QWidget" name="CreateProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="nameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="descriptionLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="startDateDayLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="startDateMonthLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="startDateYearLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="endDateDayLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="endDateMonthLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="endDateYearLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Work Amount</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Daily Reward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Chain Multiplier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Max Daily Reward</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="workAmountLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="dailyRewardLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="chainMultiplierLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="maxRewardLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newSubprojectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Subproject</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Work Amount</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="subprojectNameLineEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLineEdit" name="subprojectDateDayLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="subprojectDateMonthLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="subprojectDateYearLineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="subprojectWorkAmountLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>199</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="subprojectsLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okSubprojectButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteSubprojectButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
